specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -1616,6 +1616,10 @@
         "columnMappings": {
           "type": "object",
           "description": "Column mappings. Type: string (or Expression with resultType string)."
+        },

```

Comment:
Could you update or add an example for that?

Reply:
I meant, x-ms-example, like this:
https://github.com/iamapi/azure-rest-api-specs/tree/a3c2bff7ee233f1de44032b1fccc27450874c77f/specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/examples